tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs

```diff
@@ -58,6 +58,23 @@ private void SourceControlProviderMenuFlyout_Opening(object sender, object e)
                 ToolTipService.SetToolTip(menuItem, stringResource.GetLocalized("PrefixForDevHomeVersion", extension.PackageDisplayName));
                 menuFlyout.Items.Add(menuItem);
             }
+
+            var unassignMenuItem = new MenuFlyoutItem
+            {
+                Text = new StringResource("DevHome.Customization.pri", "DevHome.Customization/Resources").GetLocalized("MenuFlyoutUnregisteredRepository_Content"),
```

Comment:
We make the StringResource potentially a lot in this method, probably should make it once at the top of the method and re-use.

Reply:
Resolved in latest. Thanks!